### ISSUE TITLE
client_id not working for subscriber only vods, oauth_token works

### ISSUE THREAD

**Issue by Wiethoofd** [ArneVogel/concat#65]
After checking the patch from #63 I still got no valid quality options for a subscriber only VOD I do have access to by viewing directly on Twitch. (Resulted in `"error_code":"vod_manifest_restricted"` in the usher.twitch.tv response)

Some debugging and checking for the _/api/vods_ url on VOD playback in browser resulted in the confirmation that you can visit that url with `?client_id=yourclientid` as well as `?oauth_token=yourauthtoken` to get a _token_ and _sig_. The first one returns info based on set cookies, which are obviously not there on the command line using concat and the latter is personal to your account after signing in, returning a _non-null_ userid in the token.

This can be confirmed by trying to view a subscriber only VOD while logged in in browser and comparing the _/api/vods_ and usher urls being requested with `?oath_token=undefined` or your actual auth token.

Having built a version using oath_token instead of client_id actually resulted in a working subscriber-only VOD download from the command line.

**TL;DR:** _client\_id_ is not the holy grail to get vod downloads to work, using your personal _oath\_token_ is
